sway/tree/container.c

```diff
@@ -1226,10 +1226,17 @@ void container_add_gaps(struct sway_container *c) {
 		}
 	}
 
-	c->current_gaps.top = c->y == ws->y ? ws->gaps_inner : 0;
-	c->current_gaps.right = ws->gaps_inner;
-	c->current_gaps.bottom = ws->gaps_inner;
-	c->current_gaps.left = c->x == ws->x ? ws->gaps_inner : 0;
+	int gaps_horizontal = MAX(0, MIN(ws->gaps_inner, c->width - MIN_SANE_W));
+	int gaps_horizontal1 = gaps_horizontal / 2;
+	int gaps_horizontal2 = gaps_horizontal - gaps_horizontal1;
```

Comment:
I understand the idea of making this mirror the workspace gaps section, but I think the cleanness of the following outweighs it.
```c
int gaps_horizontal = MAX(0, MIN(ws->gaps_inner, c->width - MIN_SANE_W));
c->current_gaps.left = gaps_horizontal / 2;
c->current_gaps.right = gaps_horizontal - c->current_gaps.left;
```

Reply:
I had that like that once upon a time :)